python/paddle/fluid/tests/unittests/test_embeddingbag_op.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+import paddle
+import paddle.fluid as fluid
+from paddle.framework import core
+
+def manual_embeddingbag(input, params, weights = None, mode = "sum"):
+    bag = params[input]
+    if weights is not None:
+        bag *= np.expand_dims(weights, 2)
+    if mode == "sum":
+        return np.sum(bag, axis = -2)
+    else:
+        assert mode == "mean"
+        assert weights is None
+        return np.mean(bag, axis = -2)
+
+class TestEmbeddingBagCPU(OpTest):
+    def setUp(self):
+        self.op_type = "embedding_bag"
+        self.dtype = "float32"
+        self.ids_dtype = "int64"
+        self.mode = "sum"
+        params = np.random.random((32,64)).astype(self.dtype)
+        input = np.random.randint(low = 0, high = 32, size = (12,20)).astype(self.ids_dtype)
+        weight = np.random.random((12,20)).astype("float32")
+        
+        self.inputs = {'input': input, 'params': params, 'weight': weight}
+        np_out = manual_embeddingbag(input, params, weight)
+        self.outputs = {'out': np_out.reshape((12,64))}
+        self.attrs = {'mode': self.mode}
+    
+        if core.is_compiled_with_cuda():
+            self.__class__.exist_fp64_check_grad = True
+    
+    def test_check_output(self):
+        self.check_output()
```

Comment:
RuntimeError: (PreconditionNotMet) Tensor's dimension is out of bound.Tensor's dimension must be equal or less than the size of its memory.But received Tensor's dimension is 1920, memory's size is 0.